src/services/requestService.js

```diff
@@ -51,4 +52,25 @@ export default class RequestService {
   static async findRequestById(requestId, options = {}) {
     return Request.findByPk(requestId, options);
   }
+
+  /**
+   * Fetches a request instance from the database based on it's primary key within a timeframe.
+   * @static
+   * @param {sting} startDate - Request start date
+   * @param {sting} endDate - Request end date
+   * @param {integer} requestId - Primary key of the request to be fetched.
+   * @returns {Promise<string>} - Number of counts.
+   * @memberof FacilityService
+   */
+  static async search(startDate, endDate, requestId) {
+    const request = await Request.findAndCountAll({
+      where: {
+        createdAt: {
+          [Op.between]: [startDate, endDate]
+        },
+        requesterId: requestId
```

Comment:
Okay, so I realized you're passing in the user's id, please consider renaming
```suggestion
        requesterId: userId
```

Reply:
Okay